src/documents/consumer.py

```diff
@@ -408,8 +408,12 @@ def run(self) -> str:
                     mime_type = magic.from_file(self.working_copy, mime=True)
                     self.log.debug(f"Detected mime type after qpdf: {mime_type}")
                     # Save the original file for later
-                    self.unmodified_original = (
```

Comment:
I looked into this a little more, and I missed a subtle thing here.  When this path is constructed, the directory `uo` doesn't exist.  And `shutil.copy` won't create parent directories.

So the smaller fix, without creating a new temporary directory, is `self.unmodified_original.parent.mkdir(exist_ok=True)`

Reply:
Ah, thanks. Yea I thought you'd have more clarity. I changed it, thanks stumpy